ts/Grid/Core/Table/ColumnDistribution/FixedDistributionStrategy.ts

```diff
@@ -58,11 +71,40 @@ class FixedDistributionStrategy extends DistributionStrategy {
      * */
 
     public override loadColumn(column: Column): void {
-        this.columnWidths[column.id] = this.getInitialColumnWidth(column);
+        const raw = column.options.width;
```

Comment:
The variable should be meaningful, so I would suggest using "initColumnWidth" or something like that. The `raw` is too generic.

Reply:
`initColumnWidth` is confusing to me, `raw` is much better considering that context is also important. As a compromise, I propose `rawWidth`.